src/GNDStk/gnds-node.hpp

```diff
@@ -9,11 +9,8 @@ template<
    template<class...> class CHILDREN_CONTAINER
 >
 class Node {
```

Comment:
This needs to be in the file `src/GNDStk/Node.hpp` like you did with the `Tree` class.

Reply:
Yes, did this a while back as well.